cfme/services/catalogs/catalog_item.py

```diff
@@ -1,106 +1,184 @@
-# -*- coding: utf-8 -*-
-from functools import partial
 from types import NoneType
-
-from navmazing import NavigateToSibling, NavigateToAttribute
-
-from cfme.exceptions import DestinationNotFound
-from cfme.fixtures import pytest_selenium as sel
+from navmazing import NavigateToAttribute, NavigateToSibling
+from widgetastic.widget import Text, Checkbox
+from widgetastic_patternfly import Button, Input, BootstrapSelect, BootstrapTreeview
+from widgetastic_manageiq import Table, ManageIQTree
+from widgetastic.utils import VersionPick
 from cfme.provisioning import provisioning_form as request_form
-from cfme.web_ui import (
-    Form, Select, Table, accordion, fill, paginator,
-    flash, form_buttons, tabstrip, DHTMLSelect, Input, Tree, AngularSelect,
-    BootstrapTreeview, toolbar as tb, match_location, CheckboxTable)
-from utils import version, fakeobject_or_object
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
+from cfme.web_ui import tabstrip
 from utils.update import Updateable
 from utils.pretty import Pretty
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from utils import version
+
+from . import ServicesCatalogView
+
+
+class BasicInfoForm(ServicesCatalogView):
+    title = Text('#explorer_title_text')
+
+    name = Input(name='name')
+    description = Input(name='description')
+    display = Checkbox(name='display')
+    select_catalog = BootstrapSelect('catalog_id')
+    select_dialog = BootstrapSelect('dialog_id')
+    select_orch_template = BootstrapSelect('template_id')
+    select_provider = BootstrapSelect('manager_id')
+    select_config_template = BootstrapSelect('template_id')
+    field_entry_point = Input(name='fqname')
+    retirement_entry_point = Input(name='retire_fqname')
+    select_resource = BootstrapSelect('resource_id')
+    tree = VersionPick({'5.6': ManageIQTree('automate_treebox'),
```

Comment:
ManageIQTree already version picks between DynaTree and BootstrapTreeview, so I think using VersrionPick here is redundant.